test/e2e_node/g8r.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make bucket and a folder for e2e-node test logs.
+# Populate the folder from the logs stored in /tmp/_artifacts/ in the same way as a
+# jenkins build would, and then print the URL to view the test results on Gubernator
+
+bucket_name="${USER}-g8r-logs"
+echo ""
+echo "Using bucket ${bucket_name}"
+gsutil mb gs://${bucket_name}/
```

Comment:
I think it would be good to put some checks at the beginning of the script that:

1. User has gsutil
2. gcloud auth is set up correctly

Also, I think it would be good to warn the user that this script will upload the build logs (located at X) to a public bucket, and get their consent. You could add an environment variable to skip this step for experienced users.